components/TextField.vue

```diff
@@ -2,13 +2,10 @@
   <div class="flex flex-col">
     <component
       :is="tag"
-      v-bind="$attrs"
-      :class="rootClasses"
-      :value="value"
```

Comment:
I found that `:value.prop="value"` also works

Reply:
You mean for textarea?

Reply:
yes, I tried to only changed `:value="value"` to `:value.prop="value"` to solve the issue, and it works for me.

Reply:
Still not working
https://user-images.githubusercontent.com/17264716/134216207-9948ebb7-9129-4070-a2be-8d435807e09b.mov

Reply:
You may try setting a initial value for TextField rather than an empty string